modal/functions.py

```diff
@@ -1278,7 +1278,7 @@ async def remote_gen(self, *args, **kwargs) -> AsyncGenerator[Any, None]:
         if self._web_url:
             raise InvalidError(
                 "A web endpoint function cannot be invoked for remote execution with `.remote`. "
-                f"Invoke this function via its web url '{self._web_url}' or call it locally: {self._function_name}()."
+                f"Invoke this function via its web url '{self._web_url}'"
             )
```

Comment:
As there's some copy/paste here, could maybe factor this out into a `_check_no_web_url` (or whatever) helper so it's easier to keep the error message in sync.